continuous_integration/gpuci/environment.yaml

```diff
@@ -15,3 +15,30 @@ dependencies:
   - ucx-proc=*=gpu
   - ucx-py=0.27
   - xgboost=*=cuda_*
+  - dask-ml>=2022.1.22
```

Comment:
@charlesbluca I added these dependencies to the gpu script. I know we don't need them for CI but since we are using this as the place for the "dev" environment file it seemed like they were needed. You ok with this?

Reply:
My preference would be to keep these dependencies out of this file for now and use the method of updating the existing CI environment with this one, which is what the gpuCI images are currently doing:

https://github.com/rapidsai/dask-build-environment/blob/main/dask_sql.Dockerfile

If we do ever end up refactoring the image builds to do a single environment creation, then maybe it would make more sense to add these dependencies - otherwise, it just adds another environment file to keep track of that isn't being monitored by any form of CI IMO